src/combinatorics/configuration.lean

```diff
@@ -63,12 +61,12 @@ class nondegenerate : Prop :=
 (eq_or_eq : ∀ {p₁ p₂ : P} {l₁ l₂ : L}, p₁ ∈ l₁ → p₂ ∈ l₁ → p₁ ∈ l₂ → p₂ ∈ l₂ → p₁ = p₂ ∨ l₁ = l₂)
 
 /-- A nondegenerate configuration in which every pair of lines has an intersection point. -/
-class has_points extends nondegenerate P L : Type u :=
+class has_points extends nondegenerate P L :=
 (mk_point : ∀ {l₁ l₂ : L} (h : l₁ ≠ l₂), P)
```

Comment:
Maybe better to use `Π` for data fields.

Reply:
Sure, done.